src/current/cockroachcloud/upgrade-cockroach-version.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Upgrade a cluster in CockroachDB Cloud
+summary: Learn how to upgrade a cluster in CockroachDB Cloud
+toc: true
+docs_area: manage
+---
+
+This page describes how major-version and patch upgrades work and shows how to upgrade a cluster in CockroachDB {{ site.data.products.cloud }}. To upgrade a CockroachDB {{ site.data.products.core }} cluster, refer to [Upgrade to {{ site.current_cloud_version }}]({% link {{ site.current_cloud_version }}/upgrade-cockroach-version.md %}) instead.
+
+## Overview
+
+CockroachDB offers the following types of upgrades:
+
+- **Major-version upgrades**: A major-version upgrade moves a cluster from one major version of CockroachDB to another, such as from v24.2 to v24.3. A major-version upgrade may include new features, updates to cluster setting defaults, and backward-incompatible changes. Performing a major-version upgrade requires an additional step to finalize the upgrade.
+
+    As of 2024, every second major version is an **Innovation release**. For CockroachDB {{ site.data.products.standard }} and CockroachDB {{ site.data.products.advanced }}, these releases offer shorter support windows and can be skipped. Innovation releases are required for CockroachDB {{ site.data.products.basic }}, and are applied automatically.
+- **Patch upgrades**: A patch upgrade moves a cluster from one patch to another within a major version, such as from v24.2.3 to v24.2.4. Patch upgrades do not introduce backward-incompatible changes. Patch upgrades are automatically applied to CockroachDB {{ site.data.products.advanced }}, CockroachDB {{ site.data.products.standard }}, and CockroachDB {{ site.data.products.basic }} clusters.
+
+    A major version has two types of patch releases: a series of **testing releases** followed by a series of **production releases**. A major version’s initial production release is also known as its GA release. In the lead-up to a new major version's GA release, a series of Testing releases may be made available to CockroachDB {{ site.data.products.advanced }} as Pre-Production Preview releases for testing and validation. Testing releases are intended for testing and experimentation only, and are not qualified for production environments or eligible for support or uptime SLA commitments. If a cluster is upgraded to a Pre-Production Preview patch release, it will be automatically upgraded to subsequent patch releases within the major version, including newer Pre-Production Preview releases, the initial GA release, and subsequent Production patches.
```

Comment:
```suggestion
    A major version has two types of patch releases: a series of **testing releases** followed by a series of **production releases**. A major version’s initial production release is also known as its GA release. In the lead-up to a new major version's GA release, a series of Testing releases may be made available to CockroachDB {{ site.data.products.advanced }} as Pre-Production Preview releases for testing and validation. Testing releases are intended for testing and experimentation only, and are not qualified for production environments or eligible for support or uptime SLA commitments. If a cluster is upgraded to a Pre-Production Preview patch release as a major-version upgrade, it will be automatically upgraded to subsequent patch releases within the major version, including newer Pre-Production Preview releases, the initial GA release, and subsequent Production patches.
```